x-pack/legacy/plugins/security/server/routes/api/v1/authenticate.js

```diff
@@ -96,12 +96,11 @@ export function initAuthenticateApi({ authc: { login, logout }, config }, server
       const legacyConfig = server.config();
       const basePath = legacyConfig.get('server.basePath');
 
-      const nonce = await generateCSPNonce();
-      const cspRulesHeader = createCSPRuleString(legacyConfig.get('csp.rules'), nonce);
+      const cspRulesHeader = createCSPRuleString(legacyConfig.get('csp.rules'));
       return h.response(`
         <!DOCTYPE html>
         <title>Kibana OpenID Connect Login</title>
-        <script nonce="${nonce}">
+        <script>
```

Comment:
__question:__ hmmm @joshdover @kobelb aren't we effectively disabling execution of this script here assuming our default CSP rule is `script-src 'unsafe-eval' 'self';`?

Reply:
Opened an issue to discuss: https://github.com/elastic/kibana/issues/44668